pageserver/src/tenant/delete.rs

```diff
@@ -413,7 +413,7 @@ impl DeleteTenantFlow {
         let (_, progress) = completion::channel();
 
         tenant
-            .set_stopping(progress, true)
+            .set_stopping(progress, true, false)
```

Comment:
I wonder if we should add as a crutch:

```
async fn set_stopping_from_resumed_delete(&self) -> Result<_, _> {
    self.set_stopping(Default::default(), true, false).await
}
```

But well, we cannot make these foolproof.

We could add a unique load time "key" which would allow to set these, but unsure if we need that much misuse resistance.

I look forward to not having any state checkups.

Reply:
Not sure if it makes a difference. If you're in `self.set_stopping` and wondering where the parameter is true you still need to search for usages. If you're in resume delete then you'll need to skip through `set_stopping_from_resumed_delete` to get to actual impl.

Also the set of parameters changes if you resume from load or from attach. So there needs to be `set_stopping_from_delete_resumed_on_load` and `set_stopping_from_delete_resumed_on_attach`

I think I'll add a comment explaining where it is used. If you feel that helpers are better, I'll add them